.github/ISSUE_TEMPLATE/user-stories-template.md

```diff
@@ -1,22 +1,12 @@
----
-name: User Stories template
-about: The purpose of this template is to help make a description of a software feature
-  from an end-user perspective.
-title: ''
-labels: ''
-assignees: ''
+<!-- ### == <h3>, * == <ul> ,  1. == <ol> -->
```

Comment:
Why is there a comment here?

Reply:
Just to show the equivalent syntax from html to markdown so they don't have to be looked up

Reply:
Doesn't show up on the PR?

Reply:
As in, if I make a PR, will I have to remove that comment?

Reply:
No you won't have to remove it, you can leave it, it won't show up when you submit. Only show's when you edit.

Reply:
Perfect 👍 LAND HO!